### PR TITLE
#160628019 database setup

### PR DESCRIPTION
**Pull request by nwashangai**
#### What does this PR do?
it setup a postgres database
#### Description of Task to be completed?
* Install  pg module
* create migration
#### How should this be manually tested?
* Not applicable
#### Any background context you want to provide?
* None
#### What are the relevant pivotal tracker stories?
#160628019
#### Screenshots (if appropriate)
* None
#### Questions:
* None

### CODE REVIEW

File: src/app.js [hunk 1]
```diff
@@ -18,6 +19,10 @@ app.use((req, res, next) => {
   res.send({ error: '404 Sorry the page has not yet been defined try /api/v1/' });
 });
 
-app.listen(process.env.PORT || 3000, () => {});
+app.listen(process.env.PORT || 3000, () => {
+  migration().then(() => {
```
Comment (dogoyaro) [inline src/app.js:23]: Are you running your migrations any time a user connects to your app? That doesn't seem very efficient. Please explain your thought-process for this.

File: src/config/dbConnet.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import { Pool } from 'pg';
```
Comment (dogoyaro) [inline src/config/dbConnet.js]: The name of this file is spelt incorrectly. Sorry for the nitpick, I just wanted to point it out cause you might have missed it.

**Review by dogoyaro — CHANGES_REQUESTED** [review]

File: src/app.js (hunk 1 above)
Comment (nwashangai) [reply src/app.js:23]: Not anytime user connects to the app, but anytime the server is started

File: src/config/dbConnet.js (hunk 2 above)
Comment (nwashangai) [reply src/config/dbConnet.js]: ok

**Pull request merged**